src/pages/Profile/index.js

```diff
@@ -23,90 +32,361 @@ import avatar from '../../assets/avatar.svg';
 class Profile extends React.Component {
   constructor(props) {
     super(props);
+    this.ref = {
+      name: React.createRef(),
+      email: React.createRef(),
+      position: React.createRef(),
+    };
     this.state = {
-      isChecked: true,
+      dataExpireValue: '6 months',
+      editView: false,
+      displayName: 'Anisha Swain',
+      emailVal: 'xyz.redhat@gmail.com',
+      positionVal: 'Associate Software Engineer',
+      accountType: 'Pbench',
+      accountStatus: 'Active',
+      color: '#0080005e',
```

Comment:
might consider getting this data from an API request, or in worst case a `mock` file

Reply:
Added `mock` file